astropy/coordinates/tests/accuracy/test_fk4_no_e_fk4.py

```diff
@@ -20,7 +20,7 @@
 
 
 def test_fk4_no_e_fk4():
-    lines = get_pkg_data_contents('fk4_no_e_fk4.csv').split('\n')
+    lines = get_pkg_data_contents('data/fk4_no_e_fk4.csv').split('\n')
```

Comment:
Since this PR is all about consistency, maybe use `os.path.join` here too? (Same comment applies to all subsequent repeats.)

Reply:
``get_pkg_data_contents`` deals with this correctly on Windows, and in a lot of other places where we use this function we just use ``/``, presumably for readability. So I'm inclined to leave as-is?